### PR TITLE
Set up homepage banner for Best of STEM

### PR DESCRIPTION
**Pull request by kelbyhawn**
Adds a Best of Stem homepage hero banner on the code.org homepage in case we win this year's award. This is controlled by the `best-of-stem-2024` DCDO flag and we'll flip this if we win. 

## Links
Jira ticket: [ACQ-2199](https://codedotorg.atlassian.net/browse/ACQ-2199)

## Testing story
Tested locally that DCDO flag works

----

## Desktop
localhost code org_3000_(Code org - 1366x768)

| Tablet | Mobile |
| ---- | ---- |
| localhost code org_3000_(Code org - 810x1080) | localhost code org_3000_(Code org - 390x844) |

### CODE REVIEW

File: pegasus/sites.v3/code.org/homepage.json [hunk 1]
```diff
@@ -118,6 +119,20 @@
...
+    "best-of-stem-2024": {
+      "dcdo": "best-of-stem-2024",
+      "class": "best-of-stem-2024",
+      "desktopImage": "/images/banners/banner-bg-best-of-stem-confetti.png",
+      "actions": [
+        {
+          "type": "best-of-stem-2024",
+          "text_h1": "best_of_stem_heading",
+          "external_link": true,
+          "button_link": "#",
```
Comment (kelbyhawn) [inline pegasus/sites.v3/code.org/homepage.json]: I'll add this when we get the link from marketing.

**Review by bethanyaconnor — COMMENTED** [review]
Will a blog be written before we find out if we won? Wondering if we should use a DCDO flag for that

**Comment by kelbyhawn** [comment]
> Will a blog be written before we find out if we won? Wondering if we should use a DCDO flag for that

Dani asked for a link to be sent to us today (or latest by EOW) so we should be ok w/o another DCDO flag.

**Comment by bethanyaconnor** [comment]
Ah cool sounds good! Are you thinking of waiting for that link before merging this PR or updating in a separate PR?

**Comment by kelbyhawn** [comment]
> Ah cool sounds good! Are you thinking of waiting for that link before merging this PR or updating in a separate PR?

If I don't get the link by EOD today I'll merge and create a followup ticket to add the link once we get it.

File: pegasus/sites.v3/code.org/homepage.json [hunk 2]
```diff
@@ -118,6 +119,20 @@
...
+    "best-of-stem-2024": {
+      "dcdo": "best-of-stem-2024",
+      "class": "best-of-stem-2024",
+      "desktopImage": "/images/banners/banner-bg-best-of-stem-confetti.png",
+      "actions": [
+        {
+          "type": "best-of-stem-2024",
+          "text_h1": "best_of_stem_heading",
+          "external_link": true,
+          "button_link": "https://medium.com/@codeorg/code-org-wins-two-best-of-stem-2024-awards-155e55bde866",
```
Comment (kelbyhawn) [inline pegasus/sites.v3/code.org/homepage.json:132]: @bethanyaconnor just added the link!

**Review by bethanyaconnor — APPROVED** [review]

**Pull request merged**